packages/components/organisms/f-content-cards/src/components/templates/Image.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div :class="$style['c-content-cardImage--wrapper']">
+        <img
+            data-test-id="content-card-background-image"
+            :class="$style['c-content-cardImage']"
+            :src="backgroundImageUrl"
+            alt="hey">
+        <img
+            data-test-id="content-card-restaurant-logo"
+            :class="$style['c-content-cardLogo']"
+            :src="restaurantLogoUrl"
+            alt="">
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ContentCardImage',
```

Comment:
Very minor thing, but is there any reason you didn't use this as the filename? That way you can avoid including this property.

Reply:
No specific reason, I might make the change of file name to make it more specific so it's obviously associated with content cards. Will update as part of the next PR.